plugins/janus_recordplay.c

```diff
@@ -2041,6 +2064,16 @@ void janus_recordplay_update_recordings_list(void) {
 			if(e2ee)
 				rec->e2ee = TRUE;
 		}
+		if(data && data->value) {
+			rec->drc_file = g_strdup(data->value);
+			char *ext = strstr(rec->drc_file, ".mjr");
+			if(ext != NULL)
+				*ext = '\0';
+			const char *textcodec = janus_recordplay_parse_codec(recordings_path,
+				rec->drc_file, NULL, sizeof(NULL), NULL);
+			if (textcodec)
```

Comment:
See above for code style note on `if (`.

Reply:
Corrected.